core/utils/metadata-utils/src/main/java/datawave/query/model/QueryModel.java

```diff
@@ -145,27 +145,27 @@ public Collection<String> remapParameterEquation(Collection<String> projectField
                 }
             }
         }
-        
+
         return newMappings;
     }
-    
+
     /***
      * Take Current FieldName and remap it using the reverse QueryModel. Note here, If we do not find a hit in the queryModel then no aliasing is performed and
      * the default fieldname is returned. This is different behavior than in the forward case.
-     * 
-     * 
+     *
+     *
      * @param fieldName
      * @return alias
      */
     public String aliasFieldNameReverseModel(String fieldName) {
         String fName = fieldName;
-        
-        int idx = fName.indexOf('.');
-        
+
+        int idx = fName.indexOf(".");
```

Comment:
indexOf should be used with a char when searching for one character. Why changing this to string?